sale_commission/i18n/es.po

```diff
@@ -18,6 +18,12 @@ msgstr ""
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+#. module: sale_commission
+#: code:addons/sale_commission/models/sale_commission_mixin.py:55
+#, python-format
+msgid "%s commission agents"
+msgstr "%s comisionistas"
```

Comment:
Changes to .po files should be only done by weblate

Reply:
@sbidoul told me so :innocent: 

Please tell us the rule defining when we can change .po files directly and when we can not.
cc @primes2h @SimoRubi

Reply:
Trust me on this one :wink: The problem might appear if you work on an outdated base and merging Weblate translations with these ones a conflict arises, but that's detected also by Travis.